pkg/metrics/metrics.go

```diff
@@ -385,6 +385,14 @@ var (
 		Name:      "buildqueue_entries",
 		Help:      "The number of queued, waiting and running builds in the build queue",
 	}, []string{LabelBuildState, LabelBuildQueueName})
+
+	// SubroutineStart is the number of times that a subroutine has started by
+	// Cilium labeled by scope
+	SubroutineStart = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Name:      "subroutine_start_total",
+		Help:      "Number of times that a subroutine has started by Cilium labeled by scope",
+	}, []string{LabelScope})
```

Comment:
Should it be `scope` or `subsystem`?

I was thinking of scope as similar to a programming language scope:

```
scope1 {
  scope2 {
    (part of scope1 and scope 2)
  }
  (part of only scope1)
}
```